dynawo/sources/Modeler/Common/DYNSubModelFactory.cpp

```diff
@@ -39,15 +41,9 @@ SubModelFactories::SubModelFactories() {
 SubModelFactories::~SubModelFactories() {
   SubmodelFactoryIterator iter = factoryMap_.begin();
   for (; iter != factoryMap_.end(); ++iter) {
-    void* handle = iter->second->handle_;
-
-    destroy_model_t* deleteFactory = factoryMapDestroy_.find(iter->first)->second;
+    boost::function<destroy_model_t>& deleteFactory = factoryMapDestroy_.find(iter->first)->second;
```

Comment:
we could take this commit to also had some coherence in the names, 
- destroy_model_t -> deleteFactory_t
- factoryMapDestroy_ -> factoryMapDelete_

Reply:
OK